projects/dynamicforms/src/components/currency/currency.component.html

```diff
@@ -0,0 +1,30 @@
+<ng-container *ngIf="group">
+  <div class="dynamic-field" [formGroup]="group" [hidden]="config.hidden">
+    <div class="form-group">
+      <label class="col-form-label">
+        {{ label }}
+        <ng-container *ngIf="!config.required">
+          {{ 'dynamicforms.optional' | cxTranslate }}
+        </ng-container>
+      </label>
+      <div class="currency d-flex">
+        <span
+          class="currencyCode d-flex justify-content-center align-items-center"
```

Comment:
`currencyCode` should be `currency-code`, but I would name it just `currency` since you don't need that naming in wrapping class

Reply:
I need a wrapper class because I want to override  class `form-control` only in the case of currency component.

Reply:
In this case I suggest we leave only `currency` here as a class and move BS properties to it in the CSS (although `justify-content-center` wasn't needed at all). This is because it would be easier to override since different users may want currency displayed differently somehow.